android/build.gradle

```diff
@@ -36,6 +36,16 @@ android {
         minSdkVersion 24
         consumerProguardFiles 'proguard-rules.pro'
     }
+
+    compileOptions {
```

Comment:
Is this necessary? Won't this just conflict with JDK 17, 21, etc?

Reply:
It does seem necessary, couldn't figure out exactly why, but without it declared the flutter app building with this plugin says there is a conflict between the version of java used for javac and for kotlinCompile. 

With this declared the flutter app I tested with has this declared as java 17 was able to build without error. I can look into this a bit more but I set it to sdk v11 (1.8) which is the base supported by flutter.

Reply:
Java 1.8 isn't the same as Java 11 by the way

Reply:
idk what I was thinking, changed to 11